dbms/src/Storages/DeltaMerge/FilterParser/FilterParser.cpp

```diff
@@ -47,6 +47,8 @@ inline bool isRoughSetFilterSupportType(const Int32 field_type)
     case TiDB::TypeLongLong:
     case TiDB::TypeInt24:
     case TiDB::TypeYear:
+    case TiDB::TypeFloat:
```

Comment:
I think we [can't compare float directly](https://stackoverflow.com/questions/4548004/how-to-correctly-and-standardly-compare-floats). May need to modify code to support float comparison in [https://github.com/pingcap/tiflash/blob/master/dbms/src/Storages/DeltaMerge/Index/ValueComparison.h](url).